src/components/auth/torus/AuthTorus.js

```diff
@@ -101,6 +106,8 @@ export const useAlreadySignedUp = () => {
   return show
 }
 
+export const useAlreadySignedUp = isIOSNative ? useAlreadySignedUpPlaceholder : _useAlreadySignedUp
```

Comment:
```suggestion
```
this isn't needed - everything should work because:
1. we've aligned native & web interface of Torus direct SDK
2. anyway, useAlreadySignedUp consumes  existsResult provided by **our backend** so it doesn't matters which version of torus we're using
3. there're no web-specific calls inside that hook

Reply:
This is temporary, just IOS doesn't break while we wait for the torus sdk to support it